runtime/noop_jailer.go

```diff
@@ -80,4 +98,12 @@ func (j noopJailer) StubDrivesOptions() []FileOpt {
 	return []FileOpt{}
 }
 
+func (j noopJailer) Stop() error {
+	p, err := os.FindProcess(j.pid)
```

Comment:
What should we do if `j.pid` has never been set and the value is `0`? Probably want to return some sort of error?

Reply:
I agree. Fixed.